docs/docs/user-setup.md

```diff
@@ -156,6 +156,10 @@ Preparing a user config for:
 Continue? [Yn]:
 ```
 
+:::note
```

Comment:
Did you perhaps mean to add this note on line 145 instead? It makes more sense to me there.